src/Template/Loader/FractalCompoundHandlesLoader.php

```diff
@@ -129,9 +141,17 @@ private function convertToTwigPath($handle) {
 
     $path = [
       $activeTheme->getPath(),
-      reset($libs[$namespace]['paths']) . $componentName,
-      $filename . self::TWIG_EXTENSION
+      reset($libs[$namespace]['paths']),
     ];
+    if (strpos($filename, '--') === FALSE) {
```

Comment:
Please use constant `self::VARIANT_DELIMITER` here. Add newline before
```suggestion

    if (strpos($filename, self::VARIANT_DELIMITER) === FALSE) {
```